src/docs/product/alerts/alert-types.mdx

```diff
@@ -74,7 +74,7 @@ Crash rate alerts tell you when the crash free percentage for either [sessions o
 
 ### Performance
 
-For performance alerts, [metric-backed](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your alert that's not metrics-compatible, then your alert will use indexed events, or event-based data, and a message saying this will be displayed.
+For performance alerts, the data is based on [processed transaction events](/product/sentry-basics/search/searchable-properties/#processed-event-properties) data is the default. If you make a selection in configuring your alert that's not metrics-compatible, then your alert will use indexed events, or event-based data, and a message saying this will be displayed.
```

Comment:
```suggestion
For performance alerts, data based on [processed transaction events](/product/sentry-basics/search/searchable-properties/#processed-event-properties) is the default. If you make a selection in configuring your alert that's not compatible with this type of data, then your alert will use indexed events, or event-based data, and a message saying this will be displayed.
```